src/functions/smash.js

```diff
@@ -85,10 +86,14 @@ defineFunction({
         // makeVList applies "display: table-cell", which prevents the browser
         // from acting on that line height. So we'll call makeVList now.
```

Comment:
It seems like we could use a `\strut` to achieve this if we supported `\strut`.  Is that a correct way of thinking about this?

Reply:
> screenshots test

Will do.

> use a `\strut`...?

`\strut`'s serve a different purpose. They ensure that the box height does not fall below a minimum. The `table-cell` is used to prevent an out-of-control HTML line height from creating an unintended maximum.

Reply:
I see... a fraction inside a `\hphantom` might be quite tall and makes sure we clamp the height.